src/lsm/compaction.zig

```diff
@@ -207,9 +214,13 @@ pub fn CompactionType(
             const drop_tombstones = manifest.compaction_must_drop_tombstones(level_b, range);
             assert(drop_tombstones or level_b < constants.lsm_levels - 1);
 
+            const strategy: CompactionStrategy =
```

Comment:
Does the type need to be prefixed with `Compaction`?

Reply:
Seeing as we're already in the context of `CompactionType` and seeing that we also have things like `Status`?

Reply:
Its needed for the type inference on `.table_move` in the `if` expression. Another alternative is fully spelling out `CompactionStrategy.table_move` instead the anonymous enum variant; allows for type inference on the `const strategy =` then.